src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx

```diff
@@ -67,14 +110,13 @@ export default function AppointmentListItem({ appointment, facility }) {
       className="vaos-appts__card--clickable vads-u-margin-bottom--3"
       data-cy="appointment-list-item"
     >
+      {/* Disabling for now since add role=button and tab=0 fails another accessiblity check: */}
+      {/* Nested interactive controls are not announced by screen readers */}
+      {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
```

Comment:
If not this PR, we will have to bring back  `tabIndex="0"` somehow to address accessibility. Removing it will cause the UI to scroll to the bottom of the screen if veteran uses the spacebar to activate display appointment details page. 

One positive note is that using the `Return` key works as expected.